### ISSUE TITLE
Feature request: Allow user to choose which screen to use when in full-screen mode

### ISSUE THREAD

**Issue by lanroth** [MV10/monkey-hi-hat#6]
It'd be great to be able to specify which screen to use for full screen mode.

Currently it always seems to choose the same screen when switching to full screen mode. Unfortunately for me this always seems to be the wrong one. I had a look at [the code](https://github.com/MV10/monkey-hi-hat/blob/53e9a4aa5e40c512aa087a73cc8f2ac514a24dc0/mhh/mhh/Hosting/HostWindow.cs#L202)  and I think it delegates the decision to Windows. Some info on [this StackOverflow post](https://stackoverflow.com/questions/753552/going-fullscreen-on-secondary-monitor).

I did try moving the window to the secondary screen, maximizing and then fullscreening but it always seems to full screen to the other monitor. I also tried swapping the primary display in Windows to no effect.

Apologies if this is already possible.

**Comment by MV10** [comment]
Yep, this has been on my to-do list. Doing it in code last year exposed a bug in the OpenGL wrapper I use, but I think they fixed it within the past few weeks. (Since it's OpenGL and GLFW via OpenTK, it's different than the SO post, but I appreciate the link.)

**Comment by MV10** [comment]
@lanroth thanks for giving it a go!

The `--display` command only works if mhh is already fully running (not in standby), as it requires data which is only available after all the windowing stuff is created by the OpenGL windowing library (glfw).
